tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -117,3 +117,14 @@ def test__create_ids__ok(self):
 
         self.assertEqual(link_ids, get_ids(project.links))
         self.assertNotEqual(["test_project_link_id"], get_ids(project.links))
+
+
+    def test__list__ok(self):
+        generate = 5
+
+        for i in range(generate):
```

Comment:
```suggestion
        for _ in range(generate):
```
Do not need i here as it is not used.